cumulusci/cli/cci.py

```diff
@@ -600,8 +600,20 @@ def project_init(runtime):
     # Render templates
     for name in (".gitignore", "README.md", "cumulusci.yml"):
         template = env.get_template(name)
-        with open(name, "w") as f:
-            f.write(template.render(**context))
+        if not os.path.isfile(name):
+            with open(name, "w") as f:
+                f.write(template.render(**context))
+        else:
+            click.echo(
+                click.style(
+                    name
+                    + " file already exists. Please manually edit "
+                    + name
+                    + " to include:",
```

Comment:
This looks like a good opportunity to use an [f-string](https://realpython.com/python-f-strings/)

```suggestion
                    f"{name} already exists. Please manually edit {name} to include:"
```

Reply:
Also, I'd suggest this wording instead: "As a reference, here is what would be placed in the file if it didn't already exist" ... My thinking is that most of what we're generating is just reasonable defaults, and if the file already exists, there's a good chance it's been intentionally customized and our defaults wouldn't make sense.

Reply:
@davidjray Did you see my suggestion here re a different message to show the user?